shenyu-admin/src/main/java/org/apache/shenyu/admin/service/impl/SelectorServiceImpl.java

```diff
@@ -294,12 +294,14 @@ private void unbindDiscovery(final List<SelectorDO> selectors, final List<Plugin
             if (Objects.nonNull(discoveryDO)) {
                 final DiscoveryProcessor discoveryProcessor = discoveryProcessorHolder.chooseProcessor(discoveryDO.getType());
                 ProxySelectorDTO proxySelectorDTO = new ProxySelectorDTO();
+                proxySelectorDTO.setId(selector.getId());
                 proxySelectorDTO.setName(selector.getName());
-                proxySelectorDTO.setPluginName(pluginMap.getOrDefault(selector.getId(), ""));
+                proxySelectorDTO.setPluginName(pluginMap.getOrDefault(selector.getPluginId(), ""));
```

Comment:
why is  pluginMap.getOrDefault(selector.getPluginId(), "")？

Reply:
```
Map<String, String> pluginMap = ListUtil.toMap(pluginDOS, PluginDO::getId, PluginDO::getName);
```
The key for pluginMap is pluginId，the value for pluginMap is pluginName.
At this step the pluginMap's value is the divide plugin's id as key and the divide plugin'name as the value.
![image](https://github.com/user-attachments/assets/e73d8a58-dc72-41c4-8ab8-ee8c1075e721)
If use the selectorId as key, it will get null value, instead of "divide".

Reply:
thanks, great job.